js/hang-ui/README.md

```
@@ -108,3 +104,143 @@ Common components and utilities used across the package.
+3. **Wrapper generation** creates typed framework components in `src/wrappers/<framework>/`
+
+### Available Scripts
+
+```bash
+bun run prebuild   # Generate CEM + framework wrappers
+bun run build      # Build package with Vite + TypeScript declarations
+```
+
+The code generator lives in `../scripts/element-wrappers/`:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)